rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -2128,7 +2144,7 @@ static dds_qos_t * create_readwrite_qos(
     case RMW_QOS_POLICY_LIVELINESS_AUTOMATIC:
       dds_qset_liveliness(qos, DDS_LIVELINESS_AUTOMATIC, ldur);
       break;
-    case LINESS_MANUAL_BY_TOPIC:
+    case DDS_LIVELINESS_MANUAL_BY_TOPIC:
```

Comment:
Given that `qos_policies` is `rmw_qos_profile_t`, this should probably be `RMW_QOS_POLICY_LIVELINESS_MANUL_BY_TOPIC` or so.

I wonder where the original `LINESS_MANUAL_BY_TOPIC` came from.

Reply:
Thanks for the early look - this was an accidental typo that got committed while developing. I'll fix it.